Gruntfile.js

```diff
@@ -153,6 +153,7 @@ module.exports = function (grunt) {
                 '/' + staticDir + '/geo.ext.min.js',
                 // '/' + rootStaticDir + '/libs.min.js', // libs included in jade template
                 '/' + staticDir + '/jquery.gridster.js',
+                '/' + staticDir + '/jquery-ui.min.js',
```

Comment:
@aashish24 The convention right now is to copy required js/css files to a 'static' directory, including the geojs files.  I just added the jquery-ui files to the list.